css/index.css

```diff
@@ -143,4 +143,104 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+.about-page {
+    max-width: 880px;
+    width: 80%;
+    margin: 0 auto;
+}
+.middle-top {
+    border-top: 2px dotted black;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-content: stretch;
+    align-items: center;
+}
+.middle-bottom {
+    display: flex;
+}
+.quick-info {
+    margin: 20px 4%;
+    border-bottom: 2px dotted black;
+}
+.quick-info img {
+    margin-bottom: 20px;
+}
+#box1 {
+    background-color: teal;
+}
+#box2 {
+    background-color: gold;
+}
+#box3 {
+    background-color: cadetblue;
+}
+#box4 {
+    background-color: coral;
+}
+#box5 {
+    background-color: crimson;
+}
+#box6 {
+    background-color: forestgreen;
+}
+#box7 {
+    background-color: darkorchid;
+}
+#box8 {
+    background-color: hotpink;
+}
+#box9 {
+    background-color: indigo;
+}
+#box10 {
+    background-color: dodgerblue;
+}
+header {
+    align-items: center;
+    max-width: 880px;
+    display: flex;
+    width: 60%;
+    margin: 20px auto 40px;
+}
+header img {
+    width: 25%;
+}
```

Comment:
This style seems unnecessary since there are no `img` tags in your any of your `header` tags.